src/platform_impl/linux/x11/window.rs

```diff
@@ -564,9 +564,14 @@ impl UnownedWindow {
         )
     }
 
-    fn set_gtk_theme_variant(&self, variant: String) -> util::Flusher<'_> {
+    pub fn set_theme_inner(&self, theme: Option<Theme>) -> util::Flusher<'_> {
         let hint_atom = unsafe { self.xconn.get_atom_unchecked(b"_GTK_THEME_VARIANT\0") };
         let utf8_atom = unsafe { self.xconn.get_atom_unchecked(b"UTF8_STRING\0") };
+        let variant = match theme {
+            Some(Theme::Dark) => "dark",
+            Some(Theme::Light) => "light",
+            None => "",
```

Comment:
Just set `dark` for `None` I guess? Most desktop environments use dark style by default. The way to get system preference is via side channels which I'd want to see in winit.

Reply:
done, x11 and macOS will now fallback to `Theme::Dark`